packages/tests/integration/src/EntityManager.reactiveRules.test.tsx

```diff
@@ -194,7 +194,7 @@ describe("EntityManager.reactiveRules", () => {
 
   it.withCtx("runs rule added via afterMetadata", async ({ em }) => {
     // Given a publisher
-    const p = newPublisher(em);
+    const p = newPublisher(em, { authors: [{}] });
```

Comment:
Maybe could have added `authors: [{}]` as a factory default?

Reply:
that actually broke more tests than just manually inserting it in a dozen or so places 🤷